repo_policy_compliance/__init__.py

```diff
@@ -5,14 +5,21 @@
 
 from enum import Enum
 from typing import NamedTuple
-from urllib import parse
 
 from github import Github
 from github.Branch import Branch
 
+from .github_client import get_collaborators
 from .github_client import inject as inject_github_client
 
 BYPASS_ALLOWANCES_KEY = "bypass_pull_request_allowances"
+AUTHORIZATION_STRING_PREFIX = "/canonical/self-hosted-runners/run-workflows"
```

Comment:
This string might be too long to type by a human.

Reply:
The required authorization string appears in the error message asking for it, so it just needs to be copy-pasted from there. Do you think that mitigates this? It is intentionally long so that authorization isn't granted by accident

Reply:
As discussed, we will keep this for now and change it based on user feedback. One idea was to use `\`